src/Application.php

```diff
@@ -35,15 +35,16 @@ class Application extends BaseApplication
      */
     public function middleware($middleware)
     {
-        // Catch any exceptions in the lower layers,
-        // and make an error page/response
-        $middleware->push(new ErrorHandlerMiddleware());
+        $middleware
+            // Catch any exceptions in the lower layers,
+            // and make an error page/response
+            ->push(new ErrorHandlerMiddleware();
```

Comment:
Doh, the extra test files were created by me doing some baking in folder :P  I'll get the testsuite running on travis.